cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -104,6 +104,17 @@ <h1>Find a housing counselor</h1>
                                         </div>
                                     </div>
                                 </form>
+
+                                {% if zipcode and zipcode_valid %}
+                                <div class="skip-nav">
+                                    <a class="skip-nav_link
+                                              skip-nav_link__flush-left"
+                                       href="#hud_results-list_container">
+                                       Skip to results
```

Comment:
I think maybe this should be reworded. Might make the user wonder what they're skipping. "Skip map of results" maybe? /cc @niqjohnson